resources-service/migrations/20230223150400-create-resource-string-attributes-table.js

```diff
@@ -0,0 +1,120 @@
+/**
+ * Copyright (C) 2021-2023 Technology Matters
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU Affero General Public License as published
+ * by the Free Software Foundation, either version 3 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU Affero General Public License for more details.
+ *
+ * You should have received a copy of the GNU Affero General Public License
+ * along with this program.  If not, see https://www.gnu.org/licenses/.
+ */
+
+module.exports = {
+  up: async queryInterface => {
+    await queryInterface.sequelize.query(`
+      CREATE TABLE IF NOT EXISTS resources."ResourceStringAttributes"
+      (
+        "resourceId" text COLLATE pg_catalog."default" NOT NULL,
+        "accountSid" text COLLATE pg_catalog."default" NOT NULL,
+        "key" text COLLATE pg_catalog."default" NOT NULL,
+        "value" text COLLATE pg_catalog."default" NOT NULL,
+        "language" text COLLATE pg_catalog."default" NOT NULL,
```

Comment:
It specifies the 'text collation' as being the default for the DB. 'text collation' in SQL is a scheme for how strings are compared. If different columns have different collations in the DB, it could start having unexpected results when we compare strings for equality between them.

Hopefully if we keep the collation as the default everywhere it might not come up, but it might become an issue if we start trying to compare across different character sets (like Thai??)